### PR TITLE
Filter improvements and college enrollments features

### PR DESCRIPTION
**Pull request by mattharris5**
The major enhancement here is with filters:
* Filters are now saved in the url hash, so you can bookmark a set of filter settings and return to it
* Filters are now tri-state; you can show students who match a filter or students who DO NOT match a filter
* Filters are accompanied by new "stages" - currently only in use on the participants/college list to show the enrollment funnel. You can click on a stage of the pipeline to show only those students, and ALSO perform filters on that group. Everything is stored in the URL has so you can bookmark it - or create links from elsewhere in the application to pre-filter a list. For example, you might want to view only the students who have "enrolled" at UW and explore that list, so stages let you hide all the students who didn't actually enroll.

Other changes:
* The student's "most recent" college enrollment is shown on the participant detail screen, if they have any enrollments.
* Fixes a bug with the parent education level not appearing as a dropdown list
* Upgrade the rollbar gem
* Install jquery-tablesorter in preparation for using that in the future to enable client-side sorting of any table.

@mitchellharper12 Please take a look and see if this looks OK to you before I push.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -49,6 +49,7 @@ gem 'apartment-sidekiq'
 # gem 'sinatra', require: false  # for sidekiq monitor
 # gem 'slim'  # for sidekiq monitor
 gem "simple_calendar", "~> 1.1.0"
+gem 'jquery-tablesorter'
```
Comment (mitchellharper12) [inline Gemfile]: Glad this is being added, I've used it before and it's just so helpful

File: app/assets/javascripts/dreamsis.js [hunk 2]
```diff
@@ -110,5 +113,8 @@ $( function() {
     );
   })
   
+  // Enable all tablesorter tables
+  // $("table:not(.no-sort)").tablesorter()
```
Comment (mitchellharper12) [inline app/assets/javascripts/dreamsis.js]: I'm guessing this will be enabled at some point in the future?

Comment (mattharris5) [inline app/assets/javascripts/dreamsis.js]: Yes - but need to spend a little more time with it. I was hoping to globally enable it but it majorly messes up some things. Will probably just manually enable on a per-table basis. I was thinking using class name "sortable."

Also need to work on the styling a bit — but it should be a pretty easy integration.

**Comment by mitchellharper12** [comment]
Everything looks good to me, really excited to see how it looks!

**Comment by mattharris5** [comment]
Great! I will try to deploy tonight or tomorrow morning.

**Pull request merged**